tests/appsec/iast/source/test_iast_request_header.py

```diff
@@ -30,6 +34,7 @@ class TestRequestHeader:
     def setup_header(self):
         self.r = weblog.get("/iast/source/header/test", headers={"random-key": "header-source"})
 
+    @bug(context.weblog_variant == "jersey-grizzly2", reason="name field of source not set")
```

Comment:
Should we also add the library java condition? I know the weblog variant is unique for java, but adding that extra context would help readers know which library has the bug

Reply:
Weblog names are globally unique, but you're right, nothing enforce this, apart of my vigilance. so it won't hurt :)